<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XinyiFerControlTabWidget</class>
 <widget class="QWidget" name="XinyiFerControlTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1730</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="loginHistoryFrame">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>565</y>
     <width>455</width>
     <height>298</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 222, 222);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="loginHistoryFrame_horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QFrame" name="alertHistoryFrame">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>565</y>
     <width>445</width>
     <height>298</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 222, 222);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="alert_history_horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QFrame" name="state_frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>565</y>
     <width>810</width>
     <height>298</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 222, 222);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="state_frame_horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QScrollArea" name="tankScrollArea">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>1720</width>
     <height>556</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1718</width>
      <height>554</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
